client/lib/abtest/active-tests.js

```diff
@@ -37,8 +37,8 @@ module.exports = {
 	signupStepOneCopyChanges: {
```

Comment:
I think you can remove the AB test completely, and use `i18n.getLocaleSlug() === 'en'` check in place of it until translated copies are ready.